src/Patches/ModelSwaps.cs

```diff
@@ -869,23 +872,25 @@ public static void SetupItemMoveUp(Transform transform, Check check = null, Item
                     }
                 }
             }
-            moveUp.transform.localScale = TransformData.scale;
-            moveUp.transform.localPosition += new Vector3(0, 0.5f, 0);
 
-            moveUp.layer = 0;
-            moveUp.AddComponent<DestroyAfterTime>().lifetime = 2f;
-            moveUp.AddComponent<MoveUp>().speed = 0.5f;
-            moveUp.SetActive(transform.GetComponent<Chest>() != null || transform.GetComponent<TrinketWell>() != null);
             if (transform.GetComponent<SmashableObject>() != null || transform.GetComponent<DustyPile>() != null) {
                 moveUp.transform.parent = transform;
                 // so we can rotate it properly
-                if (Item.Type == ItemTypes.TRINKET) {
+                if (Item != null && Item.Type == ItemTypes.TRINKET) {
                     moveUp.name = "Card";
                 }
                 if (transform.name == "Physical Post") {
                     moveUp.transform.localScale *= 0.66f;
                 }
             }
+
+            moveUp.transform.localScale = TransformData.scale;
+            moveUp.transform.localPosition += new Vector3(0, 0.5f, 0);
+
+            moveUp.layer = 0;
+            moveUp.AddComponent<DestroyAfterTime>().lifetime = 2f;
+            moveUp.AddComponent<MoveUp>().speed = 0.5f;
+            moveUp.SetActive(transform.GetComponent<Chest>() != null || transform.GetComponent<TrinketWell>() != null);
```

Comment:
These should probably be reordered, since the moveUp's localScale is being modified then set to something else right after

Reply:
Whoops, good catch